kolibri/core/assets/src/core-getters.js

```diff
@@ -32,6 +32,10 @@ function getDefaultChannelId(channelList) {
 
 /* return the current channel object, according to vuex state */
 function getCurrentChannelObject(state) {
```

Comment:
My main thing I was trying to do was not get the error box here for having no channels, say in the first initial load of kolibri app.
https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/core-actions.js#L355
`undefined` maps to being false, I presume?
Maybe the logic for handling when there are no channels vs. when we can not find the queried channel should be different in this case.